src/lib/pages/backoffice/Acquisition/Order/OrderEditor/schema.js

```diff
@@ -196,6 +196,7 @@ export const schema = () => {
         enum: Object.values(invenioConfig.ACQ_ORDERS.statuses).map(
           (s) => s.value
         ),
+        default: Object.values(invenioConfig.ACQ_ORDERS.statuses)[0].value
```

Comment:
Why not adding a `default` field in the statuses and use that instead of the first value?